File: Online_Exam-main/Online_Exam-main/project/project/WEB-INF/classes/showall.java
import java.sql.*;
import java.io.IOException;
import java.io.PrintWriter;
import javax.servlet.ServletException;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;


public class showall extends HttpServlet
{

    
    public void doGet(HttpServletRequest request, HttpServletResponse response)
            throws ServletException, IOException
    {
    
        PrintWriter out = response.getWriter();
        
           
        

               
        out.println("<html>");
        out.println("<head>");
        
        out.println("<title>"+"Register Students"+"</title>");
		
        out.println("<style>body{background: linear-gradient(rgba(255,0,200,0.3),rgba(255,0,0,0.6)),url(admin.jpg);background-size: 100%;}");
        out.println("table{background: linear-gradient( rgba(255,255,255,0.6),rgba(255,255,255,0.6));padding: 30px;margin-top: 5%;color:black;}");
		out.println("table tr th {padding:10px}form input{background: linear-gradient( rgba(255,255,255,0.8),rgba(255,255,255,0.8));padding:15px;color:black;} form{margin-top:30px;}</style>");
		out.println("</head>");
        out.println("<body><form action='questionadd.html'><center><input type='submit' value='Back'></center></form><center><div>");
        
               
                String s1=request.getParameter("cn");
                String s2=request.getParameter("em");
                String s3=request.getParameter("an");

    
        out.println("<table border='5'>");
        
     
            
    try
               {
            Class.forName("com.mysql.cj.jdbc.Driver");
			
			String db_url="jdbc:mysql://localhost:3306/exam";
			String db_uname="root";
			String db_upass="root";
			
		Connection con=DriverManager.getConnection(db_url,db_uname,db_upass);
		
		Statement st=(Statement) con.createStatement();
                
		String q="select * from register ";
	
	      ResultSet rs;
            rs = st.executeQuery(q);
                 
            while(rs.next())

{
      out.println("<tr>");
      out.write("<th>");
      out.print(rs.getString(1));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(2));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(3));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(4));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(5));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(6));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(7));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(8));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(9));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(10));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(11));
      out.write("</th>\n");
      out.write("<th>");
      out.print(rs.getString(12));
      out.write("</th>\n");
      out.write("\n");
      out.write("</tr>\n");
      out.write("\n");
      out.write("");
           
}
    
                out.println("</th>");
            out.println("</tr>");
            
        
        out.println("</table>");

               }
    
       catch(Exception e1)
       {
           out.println(e1);
       }
         
        
    

        out.println("</div></center></body>");
        out.println("</html>");
}
}

File: Online_Exam-main/Online_Exam-main/project/project/showall.php
<html>
<head>
<html>
<head>
         <title>Question Paper</title>

         <style>
             
   	table
   	{
		background: linear-gradient( rgba(255,255,255,0.6),rgba(255,255,255,0.6));
	  	
   		padding: 100px;
   		margin-top: 5%;
                color:black; 
   	}

   	table tr td
   	{
   		font-size: 20px;
   		padding: 20px;
   	}
   	table tr td input
   	{
   		background: linear-gradient( rgba(255,255,255,0.6),rgba(255,255,255,0.6));
   		padding: 10px;
   		width: 50%;
   	}
        .m
        {
            padding: 30px; 
        }
        
                
        div ul li
        {
            float: right;
            display: inline-block;
            background-color: black;
            padding: 10px;
            margin: 8px;
         

        }
         div ul li:hover
        {
            float: right;
            display: inline-block;
            background-color:red;
			border-radius:8px;
            padding: 15px;
            margin: 3px;    
        }

body
	  {
	background: linear-gradient(rgba(255,0,200,0.3),rgba(255,0,0,0.6)),url(admin.jpg);
	background-size: 100%;
	  }



         </style>
</head>
<body>
 
<div class="m">
<form action="questionadd.html">

<center>	
  <table>
		   <center>
  		   	   <input type="submit" name="" value="Back">
  		   

<?php
$con=mysqli_connect('localhost','id11773179_root','root123','id11773179_chetan');
$sql="select * from regis";
$result = $con->query($sql);
if ($result->num_rows > 0) {
    while($row = $result->fetch_assoc()) {
    echo "<tr>";
    echo "<td>" . $row['un'] ."</td>";
    echo "<td>" . $row['fn'] . "</td>";
    echo "<td>" . $row['cn'] . "</td>";
    echo "<td>" . $row['an'] . "</td>";
    echo "<td>" . $row['em'] . "</td>";
    echo "<td>" . $row['dob'] . "</td>";
    echo "<td>" . $row['x'] . "</td>";
    echo "<td>" . $row['nal'] . "</td>";
    echo "<td>" . $row['st'] . "</td>";
    echo "<td>" . $row['pin'] . "</td>";
    echo "<td>" . $row['tq'] . "</td>";
    echo "<td>" . $row['cq'] . "</td>";
    echo "<td>" . $row['i'] . "</td>";
    echo "</tr>";
    }
}
mysqli_close($con);
?>

</table>
</center>
</form>
</div>
        
</body>
</html>